Posts/2021/03/Caps-lock-ToUpper.md

```diff
@@ -0,0 +1,206 @@
+StartsWith           Method     bool StartsWith(string value), bool StartsWith(string value, System.StringComp…
+Substring            Method     string Substring(int startIndex), string Substring(int startIndex, int length)
+ToBoolean            Method     bool IConvertible.ToBoolean(System.IFormatProvider provider)
+ToByte               Method     byte IConvertible.ToByte(System.IFormatProvider provider)
+ToChar               Method     char IConvertible.ToChar(System.IFormatProvider provider)
+ToCharArray          Method     char[] ToCharArray(), char[] ToCharArray(int startIndex, int length)
+ToDateTime           Method     datetime IConvertible.ToDateTime(System.IFormatProvider provider)
+ToDecimal            Method     decimal IConvertible.ToDecimal(System.IFormatProvider provider)
+ToDouble             Method     double IConvertible.ToDouble(System.IFormatProvider provider)
+ToInt16              Method     short IConvertible.ToInt16(System.IFormatProvider provider)
+ToInt32              Method     int IConvertible.ToInt32(System.IFormatProvider provider)
+ToInt64              Method     long IConvertible.ToInt64(System.IFormatProvider provider)
+ToLower              Method     string ToLower(), string ToLower(cultureinfo culture)
+ToLowerInvariant     Method     string ToLowerInvariant()
+ToSByte              Method     sbyte IConvertible.ToSByte(System.IFormatProvider provider)
+ToSingle             Method     float IConvertible.ToSingle(System.IFormatProvider provider)
+ToString             Method     string ToString(), string ToString(System.IFormatProvider provider), string IC…
+ToType               Method     System.Object IConvertible.ToType(type conversionType, System.IFormatProvider …
+ToUInt16             Method     ushort IConvertible.ToUInt16(System.IFormatProvider provider)
+ToUInt32             Method     uint IConvertible.ToUInt32(System.IFormatProvider provider)
+ToUInt64             Method     ulong IConvertible.ToUInt64(System.IFormatProvider provider)
+ToUpper              Method     string ToUpper(), string ToUpper(cultureinfo culture)
+ToUpperInvariant     Method     string ToUpperInvariant()
+Trim                 Method     string Trim(), string Trim(char trimChar), string Trim(Params char[] trimChars)
+TrimEnd              Method     string TrimEnd(), string TrimEnd(char trimChar), string TrimEnd(Params char[] …
+TrimStart            Method     string TrimStart(), string TrimStart(char trimChar), string TrimStart(Params c…
+```
+
+If you look carefully at this list, you can see methods that convert a string to different kinds of numbers.
+These methods would help you convert the string of 2 characters (e.g. 42) into an integer number.
+That could well be the subject of another article that shows you how to achieve this.
+
+## Summary
+
+Turning the Caps Lock key on is not something I know how to do.
+And if you did, it might confuse the user, for example if she sees the Caps Lock indicator light up on her keyboard.
```

Comment:
Use gender neutral pronouns. See https://docs.microsoft.com/style-guide/grammar/nouns-pronouns#pronouns-and-gender

```suggestion
And if you did, it might confuse the user, for example if she sees the Caps Lock indicator light up on their keyboard.
```